controllers/controllers.js

```diff
@@ -49,7 +45,13 @@ exports.getArticleIdComments = (req, res, next) => {
     selectArticleIdComments(article_id).then((comments) => {
         res.status(200).send(comments)
     })
-    .catch((err) => {
-        next(err)
+    .catch((err) =>  next(err))
+}
+
+exports.postArticleIdComment = (req, res, next) => {
+
+    insertArticleIdComment(req.params, req.body).then((comment) => {
```

Comment:
be careful sending in the params, this function now needs to be passed an object, but it really only needs the article Id